### DISCUSSION TITLE
Something's not right

### DISCUSSION THREAD

**Discussion by Scifiboy16** [community/community#46281] [New to GitHub]
### Select Topic Area

Bug

### Body

I've been trying to download Github Desktop, but instead, the download page doesn't even load. It's just stuck in a loading screen, with a message saying, 'Please click here if the page does not redirect automatically ...' and then it shows it can't reach the site. Anyone having any sort of idea about what to do in this situation, so that I could download Github Desktop again?

**Comment by P-r-e-m-i-u-m** [comment]
first @Scifiboy16  are you still there
